mad-dog/README.md

```diff
@@ -1,24 +1,20 @@
 # mad-dog
-🐶 Creator Node Test Suite
+🐶 End-to-End Protocol Test Suite (mostly covers Content Node)
 
 ## Usage
-### Standing Up/Down Services:
-1. If necessary, run `npm i` in `/mad-dog` and `/service-commands`
-2. cd to `/libs` and run `npm link`
-3. cd to `/service-commands` and run `npm link @audius/libs`
-4. Run `npm link`
-5. cd to `/mad-dog` and run `npm link @audius/service-commands`
-6. If systems are already up, run `npm run start down`
-7. Run `npm run start up`
-
-**Note:** If tests fail, sometimes running `npm i` in `/libs` and relinking packages will help. Or, bring up a fresh set of local services.
-
-Serveral tests rely on the [`pumba`](https://github.com/alexei-led/pumba), a chaos testing tool for Docker  
-**Pumba Installation**  
+### Setup/Installation -- Linking Local Packages:
+1. Make sure your services are up (`A run init-repos up && A up`). Mad dog runs against local services to provide e2e test coverage.
+2. cd to `/libs` and run `npm i && npm run build && npm link`
+3. cd to `/service-commands` and run `npm i && npm link && npm link @audius/libs`
+4. cd to `/mad-dog` and run `npm i && npm link @audius/libs @audius/service-commands`
+
+**Pumba Installation**
```

Comment:
let's actually make this pumba installation a requirement. could you update line 13 to also be included in lines 5-9?